README.md

```diff
@@ -6,7 +6,7 @@ Noir is a Domain Specific Language for SNARK proving systems. It has been design
 
 ## Quick Start
 
-Read the installation section [here](https://noir-lang.github.io/book/getting_started/nargo/installation.html).
+Read the installation section [here](https://noir-lang.org/nargo/nargo_installation/).
```

Comment:
Seems like it should be https://noir-lang.org/getting_started/nargo/nargo_installation instead.